packages/flutter_test/lib/src/matchers.dart

```diff
@@ -179,6 +195,23 @@ Matcher moreOrLessEquals(double value, { double epsilon: 1e-10 }) {
   return new _MoreOrLessEquals(value, epsilon);
 }
 
+/// Asserts that two [String]s are equal after normalizing likely hash codes.
+///
+/// A `#` followed by 5 hexadecimal digits is assumed to be a short hash code
+/// and is normalized to #000.
```

Comment:
Fair enough. Purely inertia... When this was normalizing ~9 digit codes truncating to 3 digits made more sense. switched to #00000
This will make some word wrap cases less confusing as previously word wraps weren't occurring where it would make sense due to the truncation.